build_all.sh

```diff
@@ -10,13 +10,23 @@ set -eux
 
 target=${target:-"NULL"}
 compiler=${compiler:-"intel"}
+PW_CSP=${PW_CSP:-} # TODO: This is an implementation from EPIC and consistent with the UFS WM build system.
 export MOD_PATH
 
 if [[ "$target" == "linux.*" || "$target" == "macosx.*" ]]; then
  unset -f module
  set +x
  source ./modulefiles/build.$target > /dev/null
  set -x
+#TODO: This will need to be revisited once the EPIC supported-stacks come online.
+elif [[ "${PW_CSP}" == "aws" ]]; then
+  module use /contrib/global-workflow/spack-stack/envs/ufsutils/install/modulefiles/Core
+  module load stack-intel
+  module load stack-intel-oneapi-mpi
+  module use ./modulefiles
```

Comment:
Some of these module loads look build-related. Can they be moved to the build module?

Reply:
@GeorgeGayno-NOAA Thank you for the review. I made updates related to your comments/suggestions. Please let me know if there remain issues to be addressed.